api/v1alpha1/integrationtestscenario_types.go

```diff
@@ -69,6 +69,7 @@ type TestContext struct {
 // +kubebuilder:object:root=true
 // +kubebuilder:subresource:status
 // +kubebuilder:printcolumn:name="Application",type=string,JSONPath=`.spec.application`
+// +kubebuilder:storageversion
```

Comment:
I have the feeling that we want to store the version that we want to migrate to, I suppose `v1beta1`

Reply:
Yes, `v1beta1` will be the storage version finally, but before we have the conversion to convert them and finish the controller update, we have  to use v1alpha1 as the storage version.